src/Prism.Core/Common/ParametersBase.cs

```diff
@@ -74,32 +90,72 @@ protected ParametersBase(string query)
             }
         }
 
+        /// <summary>
+        /// The count, or number, of parameters in collection
+        /// </summary>
         public int Count => _entries.Count;
 
+        /// <summary>
+        /// Returns an IEnumerable of the Keys in the collection
+        /// </summary>
         public IEnumerable<string> Keys => 
             _entries.Select(x => x.Key);
 
+        /// <summary>
+        /// Adds the <paramref name="key" /> and <paramref name="value" /> to the KeyValuePair<string,object> collection
+        /// </summary>
+        /// <param name="key">The key to reference this value in the KeyValuePair<string, object></param>
+        /// <param name="value">The value of the parameter to store</param>
         public void Add(string key, object value) =>
             _entries.Add(new KeyValuePair<string, object>(key, value));
 
+        /// <summary>
+        /// Checks collection for presense of key
+        /// </summary>
+        /// <param name="key">The key to check in the collection</param>
+        /// <returns>True if key exists; else returns false.</returns>
```

Comment:
Please use: `<c>true</c> if key exists; else returns <c>false</c>`

Reply:
I have added the reqeusted changes.  Not sure if you wanted to keep the <returns> tag but I did for now.